src/pkg/build/conveyorPacker_arch.go

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
```

Comment:
`LICENSE` in these headers should be `LICENSE.md` ... :/

Reply:
It looks like all files in 3.0 have this issue. May be better to handle this as a separate PR